gene2phenotype_project/gene2phenotype_app/templates/gene2phenotype_app/create_user_email.tpl

```diff
@@ -0,0 +1,22 @@
+{% extends "mail_templated/base.tpl" %} {% load static %}
+
+{% block subject %}
+Account Created: Welcome to G2P, {{ user }}
+{% endblock %}
+
+{% block html %}
+<p>A user account has been created using your email <strong>{{ email }}</strong> with the following details:</p>
+<ul>
+  <li><strong>First name:</strong> {{ first_name }}</li>
+  <li><strong>Last name:</strong> {{ last_name }}</li>
+  <li><strong>Username:</strong> {{ username }}</li>
+</ul>
```

Comment:
This email should include the password used to create the account. Otherwise we have to contacted them separately to let them know which password was used.

To create a new user we should generate a random password and pass it to create_user, then in the email the password is added here: <li><strong>Password:</strong> {{ password }}</li>
And we include a message asking them to change the password as soon as possible.

Reply:
That is an option.

Reply:
Can you include that info in the email, please?

Reply:
Yes working on this now